Flow.Launcher/Helper/WallpaperPathRetrieval.cs

```diff
@@ -5,44 +5,43 @@
 using System.Windows.Media;
 using Microsoft.Win32;
 
-namespace Flow.Launcher.Helper
+namespace Flow.Launcher.Helper;
+
+public static class WallpaperPathRetrieval
 {
-    public static class WallpaperPathRetrieval
-    {
-        [DllImport("user32.dll", CharSet = CharSet.Unicode)]
-        private static extern Int32 SystemParametersInfo(UInt32 action,
-            Int32 uParam, StringBuilder vParam, UInt32 winIni);
-        private static readonly UInt32 SPI_GETDESKWALLPAPER = 0x73;
-        private static int MAX_PATH = 260;
+    [DllImport("user32.dll", CharSet = CharSet.Unicode)]
+    private static extern Int32 SystemParametersInfo(UInt32 action,
+        Int32 uParam, StringBuilder vParam, UInt32 winIni);
+    private static readonly UInt32 SPI_GETDESKWALLPAPER = 0x73;
+    private static int MAX_PATH = 260;
 
-        public static string GetWallpaperPath()
-        {
-            var wallpaper = new StringBuilder(MAX_PATH);
-            SystemParametersInfo(SPI_GETDESKWALLPAPER, MAX_PATH, wallpaper, 0);
+    public static string GetWallpaperPath()
+    {
+        var wallpaper = new StringBuilder(MAX_PATH);
+        SystemParametersInfo(SPI_GETDESKWALLPAPER, MAX_PATH, wallpaper, 0);
 
-            var str = wallpaper.ToString();
-            if (string.IsNullOrEmpty(str))
-                return null;
+        var str = wallpaper.ToString();
+        if (string.IsNullOrEmpty(str))
+            return null;
 
-            return str;
-        }
+        return str;
+    }
 
-        public static Color GetWallpaperColor()
+    public static Color GetWallpaperColor()
+    {
+        RegistryKey key = Registry.CurrentUser.OpenSubKey(@"Control Panel\Colors", true);
+        var result = key?.GetValue("Background", null);
+        if (result is string strResult)
         {
-            RegistryKey key = Registry.CurrentUser.OpenSubKey("Control Panel\\Colors", true);
-            var result = key.GetValue(@"Background", null);
-            if (result != null && result is string)
+            try
+            {
+                var parts = strResult.Trim().Split(new[] {' '}, 3).Select(byte.Parse).ToList();
```

Comment:
The original has a redundant null check (because it immediately checks if it's a string after that) and then uses `.ToString()` on `object` type variable to convert it to a string even though it's already known that it's a string, just for the type system to also understand that it's a string now. I declare the string in the pattern matching if, so no need for additional conversion.